Makefile

```diff
@@ -34,7 +34,7 @@ CCASFLAGS=$(CFLAGS)
 STATIC_LDFLAGS=-lstdc++
 
 VERSION=1.6
-SHAREDLIBVERSION=2
+SHAREDLIBVERSION=0
```

Comment:
Why did you change SHAREDLIBVERSION here? This new version is not compatible with the old libraries that had SHAREDLIBVERSION=0. Please fix this (both for the openh264v1.6 branch and for master).

Also, please make a tag (v1.6.0) for the version that was used for building the binaries that were released - having a branch (where new commits can be added) isn't quite enough - there have been tags for all previous releases.